geomstats/geometry/grassmannian.py

```diff
@@ -83,4 +85,39 @@ def exp(self, vector, point):
         """
         expm = gs.linalg.expm
         mul = Matrices.mul
-        return mul(expm(vector), point, expm(-vector))
+        return mul(expm(tangent_vec), base_point, expm(-tangent_vec))
+
+    def log(self, point, base_point):
+        r"""Compute the riemannian logarithm of point w.r.t. base_point.
```

Comment:
Nit: Riemannian.

Reply:
Really? as adjective I wouldn't capitalize it would you?

Reply:
Hmm my bad it seems the usage is Riemannian in english indeed